simple-cipher/simple-cipher.js

```diff
@@ -0,0 +1,55 @@
+const alphabet = "abcdefghijklmnopqrstuvwxyz".split('');
+
+function generateNumber (min, max) {
+  var randNumber = Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
... 

Looks like a hacky/overcomplicated solution with this `randNumber === max` below